akvo/urls.py

```diff
@@ -89,6 +89,9 @@
     url(r'^activate/(?P<activation_key>\w+)/$',
         'akvo.rsr.views.account.activate', name='activate'),
 
+    url(r'^activate_invite/(?P<inviting_pk>\d+)/(?P<pk>\d+)/(?P<token_date>[0-9A-Za-z_\-]+)/(?P<token>[0-9A-Za-z_\-]+)/$',
+        'akvo.rsr.views.account.invite_activate', name='invite_activate'),
```

Comment:
The 'activate' view is directly linked to the django-registrations module, I tried to work around that since it was hard to redirect the user to a form where it can fill in a password and first and last name. That's usually already done at that point..